components/crud-web-apps/jupyter/frontend/src/app/pages/form/form-default/form-image/form-image.component.html

```diff
@@ -115,43 +117,37 @@
         {{ imageDisplayName(img) }}
       </mat-option>
     </mat-select>
-    <mat-error>
-      {{ 'jupyter.formImage.errorImageRequired' | translate }}
-    </mat-error>
+    <mat-error i18n>Please provide an Image to use</mat-error>
   </mat-form-field>
 
   <mat-form-field
     class="wide"
     appearance="outline"
     *ngIf="parentForm?.value.customImageCheck"
   >
-    <mat-label>{{ 'jupyter.formImage.customImage' | translate }}</mat-label>
+    <mat-label i18n>Custom Image</mat-label>
     <input
       matInput
-      placeholder="{{ 'jupyter.formImage.customImagePhl' | translate }}"
+      placeholder="Provide a custom Image"
```

Comment:
This placeholder also needs to be marked translation.

Reply:
pushed a commit